int32 id
float32 bearing_x
float32 bearing_y
float32 bearing_z